jobfunnel/backend/scrapers/base.py

```diff
@@ -423,6 +416,132 @@ def _validate_get_set(self) -> None:
                 [field.name for field in excluded_fields]
             )
 
+    def get_job_soups(self) -> List[BeautifulSoup]:
```

Comment:
naming here is also a bit confusing, perhaps we can call it `get_job_listings_as_soup` ? the naming somewhat conflicts with the below `_get_job_soups_page`

Reply:
Agree with the naming. It's a bit confusing as of now. Will think about more consistent and clearer naming.